src/ansys/math/core/math.py

```diff
@@ -1494,8 +1494,20 @@ def dot(self, vec) -> float:
         self._mapdl.run(f"*DOT,{self.id},{vec.id},py_val")
         return self._mapdl.scalar_param("py_val")
 
-    def asarray(self) -> np.ndarray:
-        """Return this vector as a NumPy array.
+    def asarray(self, dtype=None) -> np.ndarray:
```

Comment:
In PyMAPDL we use ``to_array`` (https://mapdl.docs.pyansys.com/version/stable/api/commands.html#ansys.mapdl.core.commands.CommandListingOutput) which makes me wonder if we should homogenise.

Numpy uses ``asarray`` so yours seems right. What do you think @RobPasMue ??

Reply:
Stick to Numpy's implementation. Users expect to see the same methods everywhere.

Reply:
If we ever get to be as famous as Numpy, then we can choos our naming 😄

Reply:
I made this change because this is the method used for the matrices. I thought it would be easier to have the same for vectors